packages/components/src/components/link/_link.scss

```diff
@@ -68,6 +68,23 @@
     font-weight: 400;
     cursor: not-allowed;
   }
+
+  .#{$prefix}--link--inline {
+    text-decoration: underline;
+
+    &:hover {
```

Comment:
The hover color is incorrect in the normal link component above (link-01 as opposed to hover-primary-text) [src](https://www.carbondesignsystem.com/components/link/style). We should fix it there rather than having it only be correct for the inline style.

Reply:
Per our style page, the default link component should also have a visited style that's being added here. It should instead be added to the regular link component unless that distinction is intentional @shixiedesign